app/assets/stylesheets/course/assessment/submission/submissions.scss

```diff
@@ -11,4 +11,14 @@
       padding: 0 3px;
     }
   }
+
+  &.index {
+    .submissions {
+      .no-submission {
+        strong {
+          color: $red;
```

Comment:
You should define a `$submissions-no-submission` variable, and define that as `#0xff0000` instead of declaring a relatively meaningless `$red`. `$red` can be declared, but the class should reference the meaningful variable instead.